### DISCUSSION TITLE
Rank decrease

### DISCUSSION THREAD

**Discussion by ArtemChuprov** [codewars/codewars.com#2613] [Q&A]
I've recently ranked up from 5 kyu to 4th. That evening i decided to go further and to get 3d kyu.
But the next morning i saw, that my progress has been removed (back to 5th). Moreover, i have now even less exp than i had before.
Does someone know, what is the problem?

**Comment by hobovsky** [comment]
As far as I can see, some of your solutions were invalidated by a moderator and points were taken away, most probably because the solutions were recognized as copied from somewhere.

Does this anwer your question? Do you need any further assistance?
